tensorflow/java/src/main/java/org/tensorflow/Output.java

```diff
@@ -22,10 +22,10 @@
  * <p>An Output is a symbolic handle to a tensor. The value of the Tensor is computed by executing
  * the {@link Operation} in a {@link Session}.
  *
- * <p>By implementing the {@link Input} interface, instances of this class could also be passed
+ * <p>By implementing the {@link Operand} interface, instances of this class could also be passed
```

Comment:
Exactly. And since I'm not even sure that the `.op` package will be included into the Android build, it might be better not to refer one of its class in the `Output` javadoc, what do you think? 
What about "also act as operands to TensorFlow operations" to use the same terminology of the `Operand` documentation?